server/src/test/java/com/cloud/network/vpc/VpcManagerImplTest.java

```diff
@@ -207,13 +208,21 @@ public void setup()
         manager._firewallDao = firewallDao;
         CallContext.register(Mockito.mock(User.class), Mockito.mock(Account.class));
         registerCallContext();
+        overrideDefaultConfigValue(NetworkService.AllowUsersToSpecifyVRMtu, "_defaultValue", "false");
```

Comment:
This is required because `NetworkServiceImplTest.java` is overriding the value which causes failures for this test file.